### PR TITLE
Create initial SKU pack for Dell C6320 and R630

### PR DESCRIPTION
**Pull request by pengz1**
Create initial SKU pack for Dell C6320 and R630.
config.json and dell specified workflows are added.

### CODE REVIEW

**Comment by pengz1** [comment]
@zyoung51 @yyscamper @RackHD/corecommitters @sunnyqianzhang

File: dell-c6320/config.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Dell R630",
```
Comment (jlongever) [inline dell-c6320/config.json]: Should this be "Dell C6320"?

**Comment by yyscamper** [comment]
:+1:

**Comment by anhou** [comment]
:+1:

**Comment by zyoung51** [comment]
+1

**Comment by sunnyqianzhang** [comment]
+1

**Pull request merged**